pkgs/development/lisp-modules/default.nix

```diff
@@ -0,0 +1,3 @@
+{ pkgs ? import <nixpkgs> { }, ... }:
+
+pkgs.callPackage ./nix-cl.nix { }
```

Comment:
What this level of indirection is used for?

Reply:
it's unneeded, removed it now in https://github.com/NixOS/nixpkgs/commit/690b62640e38bbc1e9310b3532741fbd9c6cc58c